src/index.ts

```diff
@@ -161,72 +161,75 @@ export default async (config: any, options: Props) => {
     for (const operation of operations) {
       for (const message of operation.messages()) {
         const eventType = message.extensions().get('x-eventcatalog-message-type')?.value() || 'event';
+        const isEvent = eventType === 'event';
+        const eventVisibility = message.extensions().get('x-eventcatalog-message-ownership')?.value() || 'internal';
```

Comment:
why you call it `eventVisibility` ? I mean you are reading a property called `ownership` so IMHO you should name it like `eventOwnership`.

Same for the other vars. I would go for `isOwnedByTheService` instead of `isExternal`. 

In this way we need to deal only with one concept: ownership.

Reply:
Agree